imgui_node_editor.cpp

```diff
@@ -942,24 +939,36 @@ ImLinePoints ed::Link::GetPathSameNode() const
 
 bool ed::Link::TestHitSameNode(const ImLinePoints& path, const ImVec2& point, float extraThickness) const
 {
-    auto distance = [](const ImVec2& p1, const ImVec2& p2) {
-        return ImSqrt(ImPow(p1.x - p2.x, 2) + ImPow(p1.y - p2.y, 2));
-    };
-    auto isInlLine = [&distance, &extraThickness](const ImVec2& l1, const ImVec2& l2, const ImVec2& p1) {
-        if (distance(l1, p1) + distance(l2, p1) <= distance(l1, l2) + extraThickness)
-            return true;
-        return false;
+    auto inlineDistance = [](const ImVec2& l1, const ImVec2& l2, const ImVec2& p1) {
```

Comment:
Now this function gives you the square of the distance, not the distance itself (since the square root is often useless and expensive to calculate), but below you compare it `extraThickness + m_Thickness`. 
Correct would be to compare with the square of that sum. 
```
float min_dist_sqr = (extraThickness + m_Thickness)
min_dist_sqr *= min_dist_sqr 
...
```